### ISSUE TITLE
Missing details

### ISSUE THREAD

**Issue by andrensairr** [dotnet/dotnet-api-docs#1363]
This documentation makes no mention of a type comparison between objects. The .net reference source indicates EquivalentTo will return false if `GetType() != connectionStringBuilder.GetType()`, i.e. the provided parameter is of a different type.

So, if I am comparing a SqlConnectionStringBuilder to a new DbConnectionStringBuilder object (created for the purposes of comparing the connection string), this method will return false even if an identical connection string was provided. This makes a big difference to the usefulness of the method and should certainly be documented.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f1f8754a-a889-5af6-5d72-9adeffee60d2
* Version Independent ID: 5edb5da9-79ed-6b35-adf5-2338832d9e12
* Content: [DbConnectionStringBuilder.EquivalentTo(DbConnectionStringBuilder) Method (System.Data.Common)](https://docs.microsoft.com/en-us/dotnet/api/system.data.common.dbconnectionstringbuilder.equivalentto?view=netframework-4.7.2#System_Data_Common_DbConnectionStringBuilder_EquivalentTo_System_Data_Common_DbConnectionStringBuilder_)
* Content Source: [xml/System.Data.Common/DbConnectionStringBuilder.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Data.Common/DbConnectionStringBuilder.xml)
* Product: **dotnet-api**
* GitHub Login: @douglaslMS
* Microsoft Alias: **douglasl**

**Comment by douglaslMS** [comment]
@andrensairr So a type that inherits from `DbConnectionStringBuilder` (for example, a `SqlConnectionStringBuilder`) can never be compared successfully to a `DbConnectionStringBuilder` type, regardless of whether the connection strings themselves (that is, the properties of the type) are equivalent. I believe that's what you're asking us to clarify?

**Comment by andrensairr** [comment]
@douglaslMS That is correct. It is unfortunate that that's how it was implemented, I think, but regardless  I think the documentation should mention the type comparison of the wrapping `DbConnectionStringBuilder` object.
